src/sentry/static/sentry/app/actionCreators/navigation.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import get from 'lodash/get';
+import * as ReactRouter from 'react-router';
```

Comment:
```suggestion
import {InjectedRouter} from 'react-router';
```

Reply:
@priscilawebdev VScode thinks that `InjectedRouter` isn't export from `react-router` so I did something similar:
```
import {InjectedRouter} from 'react-router/lib/Router';
```

Reply:
I understand ... I prefer not to import a module like `'react-router/lib/Router'`, because they can change folders, so we shouldn't rely on that...but we can update it when migrating to a new version

Reply:
@priscilawebdev Fair enough. `InjectedRouter` is going to be removed anyway so, either way, we will need to update this  ¯\_(ツ)_/¯